ee/vellum_ee/workflows/display/tests/workflow_serialization/generic_nodes/test_adornments_serialization.py

```diff
@@ -119,26 +115,6 @@ def test_serialize_node__retry(serialize_node):
     )
 
 
-def test_serialize_node__retry__no_display():  # GIVEN an adornment node
```

Comment:
[blocking] Why are we removing this test? Anytime we remove a test/assertions we should include rationale as to why it's warranted or lead to a user-facing regression